doc/article/en-US/templating-custom-attributes.md

```diff
@@ -104,8 +103,7 @@ Aurelia custom attributes support three different types of binding: single value
     export class SquareCustomAttribute {
       element: Element;
```

Comment:
this line isn't needed any more - the field is generated by the private modifier of the constructor argument